### ISSUE TITLE
Crash when show starts

### ISSUE THREAD

**Issue by SergioPoq** [bambuser/BambuserPlayerSDK-Android#24]
When the user is waiting for a show to start, as soon as the show begins, the app crashes

```
 Fatal Exception: java.lang.UnsatisfiedLinkError: dlopen failed: library "libbambuser.so" not found
       at java.lang.Runtime.loadLibrary0(Runtime.java:1082)
       at java.lang.Runtime.loadLibrary0(Runtime.java:1003)
       at java.lang.System.loadLibrary(System.java:1661)
       at com.bambuser.broadcaster.NativeUtils.<clinit>(SourceFile:1)
       at com.bambuser.broadcaster.MpegtsParser.<clinit>()
       at com.bambuser.broadcaster.HlsParser.<init>(SourceFile:831)
       at com.bambuser.broadcaster.BroadcastPlayer.initHlsPlayer(SourceFile:7)
       at com.bambuser.broadcaster.BroadcastPlayer$1.onResult(SourceFile:74)
       at com.bambuser.broadcaster.BroadcastPlayer$6.onPostExecute(SourceFile:2)
       at com.bambuser.broadcaster.BroadcastPlayer$6.onPostExecute(SourceFile:1)
       at android.os.AsyncTask.finish(AsyncTask.java:771)
       at android.os.AsyncTask.-$$Nest$mfinish()
       at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:788)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loopOnce(Looper.java:232)
       at android.os.Looper.loop(Looper.java:317)
       at android.app.ActivityThread.main(ActivityThread.java:8592)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
```

I am using Bambuser version 1.3.0

**Comment by SimovicIvan** [comment]
Can you provide the showId you use for reproducing this issue?

**Comment by SergioPoq** [comment]
This is the show id `PFBcAdyRXSa0B7N4meCg`

**Comment by SimovicIvan** [comment]
Fixed in [1.3.1](https://github.com/bambuser/BambuserPlayerSDK-Android/releases/tag/1.3.1)
Please note that 1.3.0 was deleted as I tried to upload a new version for 1.3.0, but that did not work.
